Description: add fennec to the browser list and change the browser preference order
Author: Fathi Boudra <fathi.boudra@nokia.com>
--- a/scripts/xdg-email
+++ b/scripts/xdg-email
@@ -699,7 +699,7 @@ fi
 if [ x"$BROWSER" = x"" ]; then
     BROWSER=links2:links:lynx:w3m
     if [ -n "$DISPLAY" ]; then
-        BROWSER=firefox:mozilla:epiphany:konqueror:chromium-browser:google-chrome:$BROWSER
+        BROWSER=chromium-browser:google-chrome:fennec:firefox:mozilla:epiphany:konqueror:$BROWSER
     fi
 fi
 
--- a/scripts/xdg-open
+++ b/scripts/xdg-open
@@ -526,7 +526,7 @@ fi
 if [ x"$BROWSER" = x"" ]; then
     BROWSER=links2:links:lynx:w3m
     if [ -n "$DISPLAY" ]; then
-        BROWSER=firefox:mozilla:epiphany:konqueror:chromium-browser:google-chrome:$BROWSER
+        BROWSER=chromium-browser:google-chrome:fennec:firefox:mozilla:epiphany:konqueror:$BROWSER
     fi
 fi
 
